eslint.config.js

```diff
@@ -102,113 +100,126 @@ export default [
             ...ts.configs.recommended.rules,
-            '@typescript-eslint/no-shadow': 'error',
-            '@typescript-eslint/no-unused-vars': 'error',
-            
+            "@typescript-eslint/no-shadow": "error",
+            "@typescript-eslint/no-unused-vars": "error",
+
             // JS specific rules
             ...eslintJsPlugin.configs.recommended.rules,
             // HACK: this eslint core rule is turned off so that the typescript-eslint version can be used instead
-            'no-unused-vars': 'off',
-            'block-scoped-var': 'error',
-            complexity: ['error', { max: 40 }],
-            'consistent-return': 'error',
-            curly: 'error',
-            'dot-location': ['error', 'property'],
-            'dot-notation': ['error', { allowPattern: '^[a-z]+(_[a-z]+)+$' }],
-            'no-alert': 'error',
-            'no-multi-spaces': 'error',
-            'no-redeclare': 'error',
-            'no-unused-expressions': ['error', { allowShortCircuit: true, allowTernary: true }],
-            'vars-on-top': 'off',
-            yoda: ['error', 'never', { exceptRange: true }],
-            'no-console': 'error', // we should use the logger instead
+            "no-unused-vars": "off",
+            "block-scoped-var": "error",
+            complexity: ["error", { max: 40 }],
+            "consistent-return": "error",
+            curly: "error",
+            "dot-location": ["error", "property"],
+            "dot-notation": ["error", { allowPattern: "^[a-z]+(_[a-z]+)+$" }],
+            "no-alert": "error",
+            "no-multi-spaces": "error",
+            "no-redeclare": "error",
+            "no-unused-expressions": [
+                "error",
+                { allowShortCircuit: true, allowTernary: true },
+            ],
+            "vars-on-top": "off",
+            yoda: ["error", "never", { exceptRange: true }],
+            "no-console": "error", // we should use the logger instead
 
             // Stylistic Issues and Opinions
-            'arrow-body-style': 'error',
-            'array-bracket-spacing': ['error', 'never'],
-            'arrow-parens': ['error', 'as-needed'],
-            'arrow-spacing': 'error',
-            'block-spacing': ['error', 'always'],
-            'brace-style': ['error', '1tbs', { allowSingleLine: true }],
-            camelcase: ['error', { allow: ['639_3'] }],
-            'comma-dangle': ['error', 'never'],
-            'comma-spacing': ['error', { before: false, after: true }],
-            'comma-style': 'error',
-            'computed-property-spacing': ['error', 'never'],
-            'function-paren-newline': ['error', 'consistent'],
-            indent: [
-                'error',
+            // arrow-body-style is not supported by @stylistic, TypeError said to use '@/arrow-body-style', so for now I have added it like that
+            "@/arrow-body-style": "error",
+            "@stylistic/quotes": ["error", "single"],
```

Comment:
I have added this line, but not sure if this was necessary.

Reply:
it should replace the quotes line further down 👍